docs/source/kedro_project_setup/configuration.md

```diff
@@ -12,6 +12,14 @@ You can also specify a source directory for the configuration files at run time
 ```bash
 kedro run --conf-source=<path-to-new-conf-directory>
 ```
+
+If you're using the [`OmegaConfigLoader`](/kedro.config.OmegaConfigLoader) you can also read configuration from a compressed file in `zip` or `tar` format. The two commands below are examples of how to reference a `tar.gz` or `zip` file.
+ ```bash
+kedro run --conf-source=/path-to-compressed-file/conf.tar.gz
```

Comment:
```suggestion
kedro run --conf-source=<path-to-compressed-file>/conf.tar.gz
```

Reply:
I think we usually specify placeholder paths or filenames in angle brackets, and then you probably don't need the forward slash.